boefjes/boefjes/plugins/kat_webpage_capture/main.py

```diff
@@ -68,51 +78,64 @@ def get_file_from_container(container: docker.models.containers.Container, path:
 
 def build_playwright_command(webpage: str, browser: str, tmp_path: str) -> str:
     """Returns playwright command including webpage, browser and locations for image, har and storage."""
-    return " ".join(
-        [
-            "npx playwright screenshot",
-            f"-b {browser}",
-            "--full-page",
-            f"--save-har={tmp_path}.har.zip",
-            f"--save-storage={tmp_path}.json",
-            webpage,
-            f"{tmp_path}.png",
-        ]
-    )
+    return [
+        "/usr/bin/npx",
+        "playwright",
+        "screenshot",
+        "-b",
+        browser,
+        "--full-page",
+        "--ignore-https-errors",
+        f"--save-har={tmp_path}.har.zip",
+        f"--save-storage={tmp_path}.json",
+        webpage,
+        f"{tmp_path}.png",
+    ]
 
 
 def run_playwright(webpage: str, browser: str, tmp_path: str = "/tmp/tmp") -> Tuple[bytes]:
     """Run Playwright in Docker."""
     client = docker.from_env()
-    res = client.containers.run(
+    client.images.pull(PLAYWRIGHT_IMAGE)
+    # https://playwright.dev/docs/docker#crawling-and-scraping
+    command = build_playwright_command(webpage=webpage, browser=browser, tmp_path=tmp_path)
+    container = client.containers.run(
         image=PLAYWRIGHT_IMAGE,
-        command=[
-            "/bin/sh",
-            "-c",
-            f"npx -y {build_playwright_command(webpage=webpage, browser=browser, tmp_path=tmp_path)}",
-        ],
+        command=command,
         detach=True,
+        ipc_mode="host",
+        user="pwuser",
+        security_opt=[
+            (
+                'seccomp={"comment": "Allow create user namespaces", "names": ["clone", "setns", "unshare"], '
+                '"action": "SCMP_ACT_ALLOW", "args": [], "includes": {}, "excludes": {}}'
+            )
```

Comment:
Maybe a comment explaining why these settings are important would be useful

Reply:
There is a comment right above the container creation: https://playwright.dev/docs/docker#crawling-and-scraping